src/Containers/FileUpload/FileUpload.stories.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { Meta, Story } from '@storybook/react';
+import {Image} from '@styled-icons/fa-solid/Image';
+import {TextLayout} from '@Layouts';
+import styled from 'styled-components'
+import { FileUpload,IFileUploadProps } from '../../index';
+import { createStoryTitle } from '../../Constants';
+
+const A=styled.a`
```

Comment:
Bottom of the file

Reply:
no, because if not error about using a variable before declaring appears.